src/cuckoo/printutil.cc

```diff
@@ -6,21 +6,21 @@
 
 namespace cuckoofilter {
 
-std::string PrintUtil::bytes_to_hex(const char *data, size_t len) {
-  std::string hexstr = "";
-  static const char hexes[] = "0123456789ABCDEF ";
+// std::string PrintUtil::bytes_to_hex(const char *data, size_t len) {
```

Comment:
Not sure why is it commented?

Reply:
I uncommented it. I tried to use cmake instead of the Makefile. I guess I did something wrong wrong on my CMakeLists.txt file, and this was a simple fix.